packages/meditrak-server/src/routes/exportSurveyResponses/exportSurveyResponses.js

```diff
@@ -123,15 +126,18 @@ export async function exportSurveyResponses(req, res) {
       }
       let exportData = getBaseExport(infoColumnHeaders).map(innerArray => innerArray.slice());
       const surveyResponseAnswers = [];
-      const processSurveyResponse = async (currentSurveyResponse, currentEntity) => {
+      const processSurveyResponse = async (currentSurveyResponse, currentEntity, nestedAnswers) => {
         const surveyDate = currentSurveyResponse.timezoneAwareSubmissionTime();
         const responseName = truncateString(currentEntity.name, 30);
         const dateString = moment(surveyDate).format(EXPORT_DATE_FORMAT);
         if (!easyReadingMode) exportData[0].push(currentSurveyResponse.id);
         exportData[1].push(currentEntity.code);
         exportData[2].push(responseName);
         exportData[3].push(dateString);
-        const answers = await findAnswersInSurveyResponse(models, currentSurveyResponse.id);
+        const answers =
+          nestedAnswers && nestedAnswers[currentSurveyResponse.id]
+            ? nestedAnswers[currentSurveyResponse.id]
+            : await findAnswersInSurveyResponse(models, currentSurveyResponse.id);
```

Comment:
Why would we want to refetch answers here? I would expect that all of them have been fetched already

If this is true, we can simply used the answers in this function's input, and remove `async` from this function's signature

Reply:
Will change 👍